orchestra/globals.py

```diff
@@ -1,4 +1,6 @@
 global loglevel
 global quiet
+global original_working_directory
 loglevel = "INFO"
 quiet = False
+original_working_directory = None
```

Comment:
I guess you're doing this to handle `orc -C somedir shell`, but I'm not sure it's what we want.
`make -C lol` or `git -C lol` usually has the semantics of `cd lol; make`.

```
$ git -C ../orchestra rebase -x pwd HEAD^^
Executing: pwd
.../revng/code/orchestra
Executing: pwd
.../revng/code/orchestra
```

Is there a situation in which you use `-C` and would like to stay in the current directory? I honestly never use `-C`.

Reply:
Yes I do use `-C` from outside orchestra directory and that's why I am proposing this change, it is frustrating to spawn a shell and having to change directory all the time.
Could we keep the current PWD if spawning a shell and change directory if executing a command?

Also, right now `-C` drops you in the build dir if that exists, otherwise in the source directory, I think we should at least be more consistent. Changing to the build directory is handy and I would like to keep it, maybe exiting with nonzero exit code if it does not exist?